<launch>
  <arg name="UAV_NAME" default="uav6" />
  <arg name="RUN_TYPE" default="real" />
  <group ns="$(arg UAV_NAME)">
    <node pkg="uhura_ros" name="uhuranode" type="uhura.py" output="screen">
      <param name="UAV_NAME" type="string" value="$(arg UAV_NAME)" />
      <param name="RUN_TYPE" type="string" value="$(arg RUN_TYPE)" />
      <param name="baudrate" type="int" value="115200" />
      <param name="debug_mode" type="bool" value="true" />
      <param name="port" type="string" value="/dev/uhura_XBEE_DEVICE_6"/>
    </node>
  </group>
</launch> 
